trappy/ftrace.py

```diff
@@ -318,6 +318,55 @@ def get_all_freqs_data(self, map_label):
 
         return ret
 
+    def run_event_callbacks(self, fn_map):
```

Comment:
Again, I'm not really convinced about this name... maybe because of its linkage with the old "compare and run" naming used for the original TRAPpy code... @sinkap up to you to proposa a more generic name for this functionality ;-)